src/checkers.cpp

```diff
@@ -19,15 +19,15 @@ ada_really_inline ada_constexpr bool is_ipv4(std::string_view view) noexcept {
     }
     last_char = view.back();
   }
-  bool possible_ipv4 = (last_char >= '0' && last_char <= '9') ||
-                       (last_char >= 'a' && last_char <= 'f') ||
-                       last_char == 'x';
-  if (!possible_ipv4) {
+  if (bool const possible_ipv4 = (last_char >= '0' && last_char <= '9') ||
```

Comment:
Why would you throw a const here?

Reply:
I feel you have something against `const` :)
If I don't see a const, I expect it to be modified later on.